types/ramda/ramda-tests.ts

```diff
@@ -668,7 +668,7 @@ interface Obj { a: number; b: number };
     const stringFunctor = {
         map: (fn: (c: number) => number) => {
             var chars = "Ifmmp!Xpsme".split("");
-            return chars.map((char) => String.fromCharCode(fn(char.charCodeAt(0)))).join("");
+            return chars.map((char) => String.fromCharCode(fn(char.charCodeAt(0)))).join("") as any;
```

Comment:
This test does not conform to the `Functor` interface, because it's `map` method does not return a functor. However I don't think this matters to this test, hence the type assertion.